doctr/models/builder.py

```diff
@@ -288,6 +290,10 @@ def __call__(
         if len(boxes) != len(text_preds) or len(boxes) != len(page_shapes):
             raise ValueError("All arguments are expected to be lists of the same size")
 
+        if not orientations:
```

Comment:
better to be specific:
```python
if not isinstance(orientations, list):
```

Reply:
Actually, better to do this:
```python
_orientations = orientations if isinstance(orientations, list) else [None] * len(boxes)
```

The `Page` constructor will handle None accordingly

Reply:
Done and pushed.